charts/linkerd2/README.md

```diff
@@ -203,24 +198,19 @@ Kubernetes: `>=1.13.0-0`
 | identity.issuer.tls.crtPEM | string | `""` | Issuer certificate (ECDSA). It must be provided during install. |
 | identity.issuer.tls.keyPEM | string | `""` | Key for the issuer certificate (ECDSA). It must be provided during install |
 | installNamespace | bool | `true` | Set to false when installing Linkerd in a custom namespace. See the [Linkerd documentation](https://linkerd.io/2/tasks/install-helmcustomizing-the-namespace) for more information. |
+| linkerdVizNamespace | string | `"linkerd-viz"` | Namespace of the Linkerd Viz Extension |
```

Comment:
I think we're only introducing this value temporarily since the controller isn't moving into the linkerd-viz extension in this PR.  I'd prefer to not introduce this and instead temporarily just hardcode the namespace used to discover the linkerd viz namespace.  We can remove this hardcoding when we move everything where it goes.